src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,53 @@
-export const TodoInfo = () => {};
+import cn from 'classnames';
+import User from '../../types/User';
+import { UserInfo } from '../UserInfo';
+import Todo from '../../types/Todo';
+import usersFromServer from '../../api/users';
+
+interface Props {
+  todo: Todo
+}
+
+export const TodoInfo: React.FC<Props> = ({
+  todo,
+}) => {
+  const {
+    id, title, completed, userId,
+  } = todo;
+
+  const user: User | null
+    = usersFromServer.find((us: User) => us.id === userId)
+    || null;
```

Comment:
null is redundant cause method filter always return a array
```suggestion
  const user: User = usersFromServer.find((us: User) => us.id === userId)
```